src/libraries/System.Private.CoreLib/src/System/Collections/Generic/EqualityComparer.cs

```diff
@@ -212,16 +210,12 @@ public override bool Equals(T? x, T? y)
     public sealed partial class ByteEqualityComparer : EqualityComparer<byte>
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public override bool Equals(byte x, byte y)
-        {
-            return x == y;
-        }
+        public override bool Equals(byte x, byte y) =>
```

Comment:
it obviously is a total nit,  and preexisting, but when using expression bodied form and it's a little expression, I'd expect to see it on the same line. (same for all these)

Reply:
I think it's more important that all such expression-body member formatting is consistent with the formatting around it.  I've now done so.